examples/run_examples.py

```diff
@@ -55,13 +56,19 @@
 req_version['saga_batch.py'] = (2019,3)
 req_version['dbscan_batch.py'] = (2019,5)
 req_version['lasso_regression_batch.py'] = (2019,5)
-req_version['elastic_net_batch.py'] = (2021,5)
+req_version['elastic_net_batch.py'] = (2022,0) # (2021,5) is relevant but there is an issue with versions number
+req_version['sycl/bf_knn_classification_batch.py'] = (2022,0) # there is an issue with versions number
+req_version['sycl/gradient_boosted_regression_batch.py'] = (2022,0) # there is an issue with versions number
 
 def get_exe_cmd(ex, nodist, nostream):
-    if not sycl_available and os.path.dirname(ex).endswith("sycl"):
-        return None
-    if req_version[os.path.basename(ex)] > daal_version:
-        return None
+    if os.path.dirname(ex).endswith("sycl"):
+        if not sycl_available:
+            return None
+        if req_version["sycl/" + os.path.basename(ex)] > daal_version:
+            return None
+    if os.path.dirname(ex).endswith("examples"):
+        if req_version[os.path.basename(ex)] > daal_version:
+            return None
```

Comment:
why is this not checked for dirs within examples, like sycl?

Reply:
There is the similar check for sycl:
if req_version["sycl/" + os.path.basename(ex)] > daal_version
I had to split checks on sycl and "classic" examples because they have the same names but different paths. Possible that it is not the best solution but I don't want to change all at the same time.